bin/check_formatting.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:async';
+import 'dart:io';
+import 'package:io/io.dart';
+
+Future<Null> main() async {
+  final ProcessManager manager = new ProcessManager();
+  Process spawn;
+
+  print('Checking config.json formatting...');
+  spawn = await manager.spawn('bin/configlet', ['lint', '.']);
+  await spawn.exitCode;
+
+  print('Checking all Dart files formatting...');
+  spawn = await manager.spawn('bin/check_formatting', []);
+  await spawn.exitCode;
+
+  await sharedStdIn.terminate();
```

Comment:
The following lines can be reused:
* print
* spawn
* await

I recommend making a function that does at least the spawn and await. It can be called twice. Once for configlet and the second for check_formatting.

Reply:
I'm aware of the code repetition and I tried to do something about it without much success. Unfortunately I haven't had much time to improve this lately. I like your suggestion though, and will do that today.